pkg/signal/signal.go

```diff
@@ -11,6 +11,11 @@ import (
 	"syscall"
 )
 
+const (
+	// DefaultStopTimeout is the timeout (in seconds) for the syscall signal used to stop a container.
+	DefaultStopTimeout = 10
```

Comment:
Ah, yes, if possible, I prefer the CLI to not have to send anything if it's the default